Lambdas/GHOAuth/Models/User.swift

```diff
@@ -0,0 +1,4 @@
+struct User: Decodable {
```

Comment:
```suggestion
struct User: Codable {
```
(If only because there's no reason _not_ to.)